client/state/selectors/get-user-devices.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { get, values } from 'lodash';
+
+/**
+ * Returns the current user's devices
+ *
+ * @param  {Object} state Global state tree
+ * @return {Object}       current user's devices
+ */
+export default function getUserDevices( state ) {
+	return values( get( state, 'userDevices.items' ) );
+}
```

Comment:
I was working on that change 😉 , now on beb8d8f06ef968f29cc0018d5c04d2a2d2cdccee.